apps/wing/tsconfig.json

```diff
@@ -22,6 +22,7 @@
     "declaration": true,
     "baseUrl": ".",
     "esModuleInterop": true,
+    "types": ["@withfig/autocomplete-types"]
```

Comment:
How is this used?

Reply:
This is needed for `Fig.Arg` and `Fig.Option`

Reply:
You should be able to do something like `import type { Arg, Option } from "@withfig/autocomplete-types"` (you may have to alias them) instead of adding the type here.

Reply:
I also tried, pnpm build works with this, but then the command `node dist/cli.js generate-fig-spec >> test.ts` does not find `@withfig/autocomplete-types` 😕
So I found this https://github.com/withfig/autocomplete-tools/blob/main/types/README.md and it worked. I also don't like the compilerOptions param, but didn't find a workaround.